rust/src/mlil/mod.rs

```diff
@@ -0,0 +1,21 @@
+use std::fmt;
+
+use crate::function::Location;
+use crate::types::Variable;
+
+mod block;
+mod expression;
+mod function;
+mod info;
+mod lift;
+mod operation;
+
+pub use self::block::*;
+pub use self::expression::*;
+pub use self::function::*;
+pub use self::info::*;
+pub use self::lift::*;
+pub use self::operation::*;
+
+pub use self::block::Block as MediumLevelBlock;
```

Comment:
Instead of aliasing them, just rename them where they're defined. It'll make maintaining this a little easier not having to remember the duality of the naming.